src/app/(explore)/speakers/page.tsx

```diff
@@ -0,0 +1,16 @@
+"use client"
```

Comment:
I think this page should also be server rendered

Reply:
same issue as https://github.com/bitcointranscripts/registry/pull/43#discussion_r1795509650